modules/util/portal-tools-rest-builder/src/main/resources/com/liferay/portal/tools/rest/builder/dependencies/base_resource_test_case.ftl

```diff
@@ -1359,12 +1359,15 @@ public abstract class Base${schemaName}ResourceTestCase {
 					/>
 
 					<#if freeMarkerTool.hasPostSchemaJavaMethodSignature(javaMethodSignatures, firstPathJavaMethodParameter.parameterName, schemaName) && stringUtil.equals(javaMethodSignature.methodName, "post" + modifiedPathJavaMethodParameterName + schemaName)>
```

Comment:
What's the difference between `schemaName` and `schemaNames`?

Reply:
@4lejandrito the difference between `schemaName` and `schemaNames` is the same name but `schemaNames` is in plural:
- `schemaName`: `Diagram` 
- `schemaNames`: `Diagrams`